src/ast.rs

```diff
@@ -109,12 +109,15 @@ impl Visitable for Text<'_> {
 #[derive(Debug)]
 pub struct Escape {
   pub start: Location,
-  pub escaped_char: char,
+  pub escaped_char: Option<char>,
```

Comment:
This is very annoying, because when formatting a message you always need to read in escapes and, if the message is valid, this is actually never None.

Maybe can we format the `\` at the end of the file as a Text rather than Escape, and leave this char as non-optional?

Reply:
We can also just not emit a token for it at all

Reply:
Yes that works too